File: FindTheHighestScore/py3/fths.py
# find the highest score solution in py2 for code eval by steven a dunn

import sys
from operator import itemgetter

f = open(sys.argv[1], 'r')
for line in f:
	scores = []
	line = line.split(" | ")
	for i in range(len(line)):
		scores.append(list(map(int, line[i].split())))
	scores = list(zip(*scores))
	print " ".join([str(max(i)) for i in scores])
f.close()

File: FindTheHighestScore/cpp/fths.cpp
// find the highest score soln in c++ for code eval by steven a dunn

#include <fstream>
#include <iostream>
#include <string>
#include <vector>

using std::cout;
using std::endl;
using std::ifstream;
using std::string;
using std::vector;

vector<int> tokenize(string);

int main (int argc, char* const argv[])
{
	ifstream input_file(argv[1]);
	string line;
	
	if(input_file)
	{
		while (getline(input_file, line))
		{
			vector<int> tokenize(line);
		}
		input_file.close();
	}
	return 0;
}

vector<int> tokenize(string input)
{
	vector<int> dummy;
	return dummy;
}